pkg/api/handlers/compat/containers_logs.go

```diff
@@ -152,9 +152,7 @@ func LogsFromContainer(w http.ResponseWriter, r *http.Request) {
 		}
 
 		frame.WriteString(line.Msg)
-		// Log lines in the compat layer require adding EOL
-		// https://github.com/containers/podman/issues/8058
-		if !utils.IsLibpodRequest(r) {
+		if !line.Partial() {
```

Comment:
Does this match docker?

Reply:
We use the same handler for both types of clients, so this makes both behave the same.  Anecdotally, when I pointed a `docker` client at the podman socket, I get what looks like the correct behavior on the client end.